app/views/News.js

```diff
@@ -127,16 +100,38 @@ class NewsScreen extends Component {
               style={{
                 backgroundColor: Colors.VIOLET_BUTTON_DARK,
                 flex: 1,
-                paddingVertical: 16,
+                flexDirection: 'row',
+                alignItems: 'center',
+                justifyContent: 'center',
+                paddingHorizontal: 16,
               }}>
-              <Carousel
-                data={this.state.newsUrls}
-                renderItem={this.renderItem}
-                sliderWidth={width}
-                itemWidth={width * 0.85}
-                layout={'default'}
-                scrollEnabled
-              />
+              <ScrollView style={{ width: width * 0.85 }}>
+                {this.state.newsUrls.map((item, index) => (
+                  <View key={index} style={styles.singleNews}>
```

Comment:
It feels like we can pull this out into a standalone component to clean things up a bit, e.g. 

```js
{this.state.newsUrls.map((item, index) => {
  return <NewsItem item={item} index={index} />
}
```